src/bc.d

```diff
@@ -31,7 +31,7 @@ enum ShortInst : ubyte
     Neg,
     Soh, /// set hi stack addr
     Sdh, /// set hi dataseg addr
-
+    Flg, // writes the conditionFlag into [lw >> 16]
```

Comment:
not really a need. 
I just like them because that is what asm looks like.
If you have better names please suggest them.